library/alloc/src/boxed.rs

```diff
@@ -1364,6 +1364,38 @@ impl<A: Allocator> Box<dyn Any + Send, A> {
     }
 }
 
+impl<A: Allocator> Box<dyn Any + Send + Sync, A> {
+    #[inline]
+    #[stable(feature = "XXX", since = "1.51.0")]
```

Comment:
What should be done about the feature name here?

Reply:
What do you think about "box_send_sync_any_downcast"?

Reply:
Thanks, let's go with that then :)